mw-webapp/src/component/notification/Notification.tsx

```diff
@@ -0,0 +1,69 @@
+import {ReactElement} from "react";
+import {Cross2Icon} from "@radix-ui/react-icons";
+import {Close, Description, Provider, Root, Viewport} from "@radix-ui/react-toast";
+import styles from "src/component/notification/Notification.module.scss";
+
+/**
+ * Control the sensitivity of the toast for accessibility purposes.
+ * Foreground - for toasts that are the result of a user action.
+ * Background - for toasts generated from background tasks .
+ */
+export enum NotificationType {
+  "foreground" = "foreground",
+  "background" = "background",
+}
+
+/**
+ * Notification props
+ */
+interface NotificationProps {
+
+  /**
+   * The time in milliseconds that should elapse before automatically closing each toast.
+   * @default 5000
+   */
```

Comment:
it's default value of radix-ui. I didn't change it. 
if someone reads slowly or there will be a big notice. This value can be changed.

Reply:
I think it's wrong. We as developers can't know who will read notification and how big it will tomorrow. So how we will change duration? In which cases? I think better don't use auto closing notification as common.
But if it's  a design concept then ok but then this property no need at all (make it with same duration at all).
We  can speak about it in the group all together, write there pls